src/graphqlProtocol/ProtocolGraphQLConfiguration.ts

```diff
@@ -262,6 +274,71 @@ export class ProtocolGraphQLConfiguration extends ProtocolConfiguration {
     const typeName = this.generateEntityTypeName(entity);
     return generateTypeNamePascalCase(`${typeName}-connection`);
   }
+
+  generateSubscriptionInstanceInputTypeName(entity, subscription) {
```

Comment:
Lots of code duplicated from mutation, maybe it could be nice to refactor by merging mutation and subscription naming functions ?

Reply:
this part needs a heavy review anyway. 
like already mentioned once I'd love to make it more accessible to devs to make changes easier for each project.
I'm open for ideas